src/main/java/com/shaft/tools/io/ReportManager.java

```diff
@@ -19,706 +19,728 @@
 
 public class ReportManager {
```

Comment:
Please do not restyle the entire class file (just the lines you've added) as GitHub is detecting that it's all new.